File: example_a/src/main/java/cn/figo/example_a/MyApplication.java
package cn.figo.example_a;

import android.app.Application;

/**
 * @author Weir.
 * @date 2019/2/22.
 */
public class MyApplication extends Application {

    @Override
    public void onCreate() {
        super.onCreate();
        ExampleAInterface.init(this);
    }
}

File: app/src/main/java/cn/weir/simplecomponent/MyApplication.kt
package cn.weir.simplecomponent

import android.app.Application
import cn.figo.example_a.ExampleAInterface
import cn.weir.base.SimpleComponentInterface

/**
 * @author Weir.
 * @date   2019/2/13.
 */
class MyApplication : Application() {

    override fun onCreate() {
        super.onCreate()

        SimpleComponentInterface.setDebugMode(true)
        SimpleComponentInterface.init(this)

        //Example A 模块
        ExampleAInterface.init(this)
    }

}